DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/Services/CachingService.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Collections.Generic;
+using System.Collections;
+
+namespace Speckle.Converters.RevitShared.Services;
+
+public class CachingService : ICachingService
```

Comment:
What is this caching? Is this pre-existing? We have caching in a few places so... just checking where/what

Reply:
It's also an interesting class and I am curious about the Funcs being added

Reply:
yes this is existing. We do have caching in a ton of places. This was my attempt to bring all the caching into one object. It works well (I think) but we still have caching in a lot of places that we haven't moved over.